### DISCUSSION TITLE
Difficulty level or tier requirement for turrets

### DISCUSSION THREAD

**Discussion by BlackMoons** [FakeFishGames/Barotrauma#14523] [Ideas and suggestions]
Can we have some kind of difficulty level (the 0~100% thing) or tier requirements for turrets?

Lots of mods add new turrets, but they are almost always way better then vanilla. This would be fine if they where late game options, but I can't find any way to lock turrets behind level difficulty or submarine tier? 

I think I would like level difficulty for turrets more, so they are in line with the level restrictions we have for all other items.

**Comment by Ek-Videogames** [comment]
A cool thing for modding perhaps, and a good workaround for the potential issue of hardpoints being a bit 'too generalized' in some cases; 
but I'd say doing so in Vanilla would stratify the already thin variety of turrets too much and basically eliminate choice entirely or limit it to 2 options. ( just like how tier restrictions did to the very tiny gamut of vanilla submarines ).

Seeing as the game is kind of 'mostly done development' I don't exactly expect the devs are going to do any big content update that essentially triples the number of turret options in the game.

**Reply by BlackMoons** [reply]
I am fine with all the vanilla turrets being unlocked in the 1st or 2nd biome TBH. This is more a feature I wanna see for modders to enjoy. 

It would however be interesting to see some 'direct upgrade' turrets at some point. Even things like 'heavy coilgun' that is just a faster firing coilgun or whatever you could buy in the 3rd biome for $$$.
